src/app/item-comments/item-comments.component.ts

```diff
@@ -12,6 +12,8 @@ import { HackerNewsAPIService } from '../hackernews-api.service';
 export class ItemCommentsComponent implements OnInit {
   sub: any;
   item;
+  pollResults: any[] = [];
+
```

Comment:
I know we're retrieving the poll results individually async, but I'm okay with that for now. However I think it'll be nicer if the poll results show in the view only when the entire pollResults array is filled (instead of one by one as they load).

So I think we can do this by shifting the local variable `howManyPollResults` to the component level here (just declare it after `pollResults` as type number (`howManyPollResults: number`). And then using it later as `this.howManyPollResults` instead of locally declaring it.

Then in our view we can just add a length check in the `ngIf` condition.